tests/phpunit/tests/block-supports/typography.php

```diff
@@ -3,7 +3,9 @@
  * @group block-supports
  */
 class Tests_Block_Supports_Typography extends WP_UnitTestCase {
-
+	/**
+	 * @covers ::wp_apply_typography_support
```

Comment:
Are you able to add ticket numbers to this and any other test missing them too?

`@ticket` is an alias for `@group` so can be super handy when wanting to test a single ticket.

Reply:
Ah for sure. I'll add ticket refs for the new tests, and try to get it right for the older ones if I can find them. 🍺